src/Containers.jsx

```diff
@@ -645,17 +645,20 @@ class Containers extends React.Component {
             // Remove infra containers
             filtered = filtered.filter(id => !this.props.containers[id].IsInfra);
 
+            const getHealth = id => {
+                const state = this.props.containers[id]?.State;
+                return state?.Health?.Status || state?.Healthcheck?.Status;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1540-20240128-084429-20c044a7-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)